frontend/actions.js

```diff
@@ -95,23 +95,30 @@ export default ({setState, getState}) => {
     }
   }
 
-  const validateSendAddress = (state) => setState({
-    sendAddress: Object.assign({}, state.sendAddress,
-      {validationError: !Cardano.isValidAddress(state.sendAddress.fieldValue) ? {code: 'SendAddressInvalidAddress'} : ''}),
-  })
+  const validateSendAddress = (state) =>
+    setState({
+      sendAddress: Object.assign({}, state.sendAddress, {
+        validationError: !Cardano.isValidAddress(state.sendAddress.fieldValue)
+          ? {code: 'SendAddressInvalidAddress'}
+          : '',
+      }),
+    })
 
   const validateSendAmount = (state) => {
     let validationError = ''
```

Comment:
I know this is an older design decision but I am a not big fan of polymorphism of `string` vs `dict`. Is there a good reason not to use `null` for non-error scenario?

Reply:
there isn't. I prefer nulls, too, so I changed it. Fixed